android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIDialog.java

```diff
@@ -335,7 +335,7 @@ public void onCancel(DialogInterface dlg)
 			// can also be used.
 			ListView listView = dialog.getListView();
 			if (listView != null) {
-				listView.setContentDescription(composeContentDescription());
+				listView.setContentDescription(getProxy().composeContentDescription());
```

Comment:
I see we are validating if the proxy is available when checking for the `accessibilityHidden` property, maybe we should do that here, too?